alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -230,6 +230,10 @@ class AlertIndices(
         return alertIndexInitialized && alertHistoryIndexInitialized
     }
 
+    fun isAlertIndexInitialized(): Boolean {
```

Comment:
This checks the alerts index and not the alert config index. We need a different function that checks if the config index is initialized.

Reply:
got it, updated